welcome/learn_more_about_openshift.adoc

```diff
@@ -303,20 +301,38 @@ a|* xref:../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-sched
 * xref:../machine_management/creating-infrastructure-machinesets.adoc#creating-infrastructure-machinesets[Creating infrastructure machine sets]
 |===
 
+[id="self-managed-hcp"]
+== {hcp-capital}
 
-//
-//[id="hosted-control-plane-activities"]
-//== Hosted control plane activities
-
-//[options="header",cols="2*"]
-// |===
-// |Learn about {hcp-capital} |Optional additional resources
-
-// | xref:../hosted_control_planes/index.adoc#hosted_control_planes[Hosted control planes overview]
-// | xref:../hosted_control_planes/index.adoc#hosted-control-planes-overview_hcp-overview[Introduction to hosted control planes]
-
-// | xref:../hosted_control_planes/hcp-getting-started.adoc#hcp-getting-started [Getting started with hosted control planes]
-// |
-
-// |===
-//
\ No newline at end of file
+[options="header",cols="2*"]
+|===
+|Learn about {hcp} |Optional additional resources
+
+| xref:../hosted_control_planes/index.adoc#hosted-control-planes-overview[Hosted control planes overview]
+a|
+xref:../hosted_control_planes/index.adoc#hosted-control-planes-version-support_hcp-overview[Versioning for {hcp}]
+
+| Preparing to deploy
+a| * xref:../hosted_control_planes/hcp-prepare/hcp-requirements.adoc#hcp-requirements[Requirements for {hcp}]
+* xref:../hosted_control_planes/hcp-prepare/hcp-sizing-guidance.adoc#hcp-sizing-guidance[Sizing guidance for {hcp}]
+* xref:../hosted_control_planes/hcp-prepare/hcp-override-resource-util.adoc#hcp-override-resource-util[Overriding resource utilization measurements]
+* xref:../hosted_control_planes/hcp-prepare/hcp-cli.adoc#hcp-cli[Installing the {hcp} command-line interface]
+* xref:../hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc#hcp-distribute-workloads[Distributing hosted cluster workloads]
+* xref:../hosted_control_planes/hcp-prepare/hcp-enable-disable.adoc#hcp-enable-disable[Enabling or disabling the {hcp} feature]
+
+| Deploying {hcp}
```

Comment:
A lot of links exist for HCP. Linking to the Deploying hosted control planes  level 1 might be difficult but it would cut down the number of links for HCP. A generalized link like "Deploying hosted control planes on supported platforms" would be a nice future update.

Reply:
cc @lahinson FYI

Reply:
Thanks for the mention, @opayne1. I'm okay with implementing @dfitzmau's suggestion, but I'm not exactly sure how to form the xref since we would be pointing to essentially a topic map entry (i.e., the top level "Deploying HCP" content doesn't have a .adoc file. On docs.redhat.com, the link we'd want to point to is https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/hosted_control_planes/deploying-hosted-control-planes. I would need to play around with the xref syntax to figure out what the exact xref should be.

Reply:
You can't xref to that location - you'd need to create a new overview topic of some kind. I think you should leave this as it is.

Reply:
Thanks for confirming that, Kathryn -- and for saving me a lot of time trying to make an xref work!